src/v/kafka/server/connection_context.h

```diff
@@ -45,6 +45,7 @@ class connection_context final
       , _sasl(std::move(sasl))
       // tests may build a context without a live connection
       , _client_addr(_rs.conn ? _rs.conn->addr.addr() : ss::net::inet_address{})
+      , _client_port(_rs.conn ? _rs.conn->addr.port() : 0)
```

Comment:
Why is this a materialized member variable. Why not make the client_port() a function?

Reply:
that seems like a good improvement

Reply:
I can make that change in my PR